presubmit_tests/test_pr.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### Transferring shell commands used in https://gist.github.com/jcnars/26b8526cde45a1bb5778a36abe90b96b
+### into this test script
+### context / reference: (internal) at: http://b/202240337#comment22
+
+# set up ssh from pod to BMX host
+# using sydney for initial testing
+mkdir /root/.ssh
+chmod 0700 /root/.ssh
+ssh-keyscan -tecdsa 172.16.30.1 > /root/.ssh/known_hosts
+
+# install pre-reqs
+pip install jmespath
+cp /etc/files_needed_for_tk/google-cloud-sdk.repo /etc/yum.repos.d/google-cloud-sdk.repo
+yum install google-cloud-sdk -y
+
+# run the cleanup script
+cd ..; ./cleanup-oracle.sh --ora-version 19 --inventory-file /etc/files_needed_for_tk/nonrac-inv --yes-i-am-sure --ora-disk-mgmt udev --ora-swlib-path /u01/oracle_install --ora-asm-disks /etc/files_needed_for_tk/nonrac-asm.json --ora-data-mounts /etc/files_needed_for_tk/nonrac-datamounts.json
```

Comment:
I'd prefer an absolute path here

Given that you have most of the install and cleanup configuration, you might want to put the asm disk and other files here too, to avoid splitting test configuration over too many places

Reply:
I would have gone with absolute path if I am manually doing a `git clone` of the PR.
But since the prow is responsible for this clone, I don't know at this point at what location it will clone it. and hence I am not able to put the absolute path. We will change it to absolute once we do our first prowjob runs.

Reply:
Where does stuff like nonrac-datamounts.json come from?  I think it might make sense to put the command-line arguments to cleanup/install-oracle.sh and the json files in the same place (this script, perhaps, or at least same directory/PR?).

And anit:  Can you use \ line continuation characters to keep line length reasonable?

Reply:
The `nonrac-datamounts.json` file comes from the GKE build cluster in `bmaas-testing` project's K8s configmaps.

https://github.com/GoogleCloudPlatform/oss-test-infra/blob/375bd5042646c04aff266a2bb82c397e18e5afeb/prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml

has the details for files like `nonrac-datamounts.json` f, `nonrac-asm.json`, etc.
They all get mounted in the same place in the pod under: `/etc/files_needed_for_tk/`

(Internal) https://b.corp.google.com/issues/202240337#comment12 has the configmap creation steps.

2 points on the location of the `*.json` files:
1) They get mounted when the pod itself is created by our GKE build cluster
2) The cloning of the PR-ed repo happens after pod creation and hence it looks like at this point (before the first run), they cannot be combined in the same place.

As noted in previous comment, at this point, it's unclear where prow will clone the PR-ed repo. Once we have the first run, we can attempt to put everything in same place.

Reply:
Added line break \ to keep it readable